app/views/admin/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<section id="admin">
+  <div class="container">
+    <h1 class="center">Admin Panel</h1>
+
+    <section>
+      <h3 class="center">Active Admins</h3>
+      <div class="admin-wrapper">
+        <% User.all.each do |u| %>
+          <% if u.admin == true %>
+            <div class="content-wrapper">
+              <h5><%= u.first_name %> <%= u.last_name %></h5>
+              <h6><%= u.email %></h6>
+              <% @user = u %>
```

Comment:
Setting instance variables inside views is generally a bad idea.  Instead, I'd reference it by `u`.